docs/src/main/tut/codecs/custom-codecs.md

```diff
@@ -132,7 +132,7 @@ implicit val config: Configuration = Configuration.default
 Bar(13, "Qux").asJson
 ```
 
-It's worth noting that if you don't want to use the experimental generic-extras module, the
+It's worth noting that if you don't want to use the `generic-extras` module, the
```

Comment:
Same as the other note.  Want people to consider it experimental, know that it may not be supported in exactly the same way in the future.